assets/src/edit-story/app/media/media3p/api/media3pApiProvider.js

```diff
@@ -189,11 +189,26 @@ function Media3pApiProvider({ children }) {
     };
   }
 
+  /**
+   * Register usage of a media for the given payload.
+   *
+   * @param {Object} obj - An object with the options.
+   * @param {string} obj.payload The payload to be posted to register the usage.
+   * @return {Promise<Object>} An object with the category
+   * resources.
+   */
+  async function registerUsage({ payload }) {
+    await apiFetcher.registerUsage({
+      payload,
```

Comment:
[nit]: This can be written on one line:

await apiFetcher.registerUsage({ payload });

Reply:
Done.